nav2_collision_monitor/src/collision_monitor_node.cpp

```diff
@@ -378,6 +397,36 @@ void CollisionMonitor::process(const Velocity & cmd_vel_in)
     }
   }
 
+  if (collision_points_marker_pub_) {
+    // visualize collision points with markers
+    visualization_msgs::msg::MarkerArray marker_array;
+    if (collision_points.size() > 0) {
+      visualization_msgs::msg::Marker marker;
+      marker.header.frame_id = get_parameter("base_frame_id").as_string();
+      marker.header.stamp = rclcpp::Time(0, 0);
+      marker.ns = "collision_points";
+      marker.id = 0;
+      marker.type = visualization_msgs::msg::Marker::POINTS;
+      marker.action = visualization_msgs::msg::Marker::ADD;
+      marker.scale.x = 0.1;
```

Comment:
10cm seems a little big, don't you think?

Reply:
yep agree, reducing to 0.2